src/components/payment/staff-payment.vue

```diff
@@ -100,4 +122,14 @@ export default class StaffPayment extends Vue {
 ::v-deep .v-input--radio-group__input {
   background-color: white;
 }
+
+// remove margin below radio group
+::v-deep .v-input--radio-group > .v-input__control > .v-input__slot {
+  margin-bottom: 0 !important;
```

Comment:
Ideally, this (and the rules on line 116 and 132) should be in the shared component itself, but I didn't want to destabilize its use in other UIs.